draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -742,6 +742,10 @@ Plain Text
 ECDHE-SHA256-C25519
 : Public Key Material is exchanged via ECDHE key exchange with SHA256 and
   Curve 25519 to generate the derived key (d-key) from the shared secret.
+  The full potential of ECDHE use is realized when it is combined with peer
+  authentication technologies to thwart men-in-the-middle attacks. This can
```

Comment:
/thwart/ seems an odd word, is /protect against/ a better sense of what is intended?

Reply:
Solved in https://github.com/tsvwg/ietf-multipath-dccp/pull/255/commits/4cbdf12d3696ebdcea4fd0064ead62931e1d10cf